extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduled.java

```diff
@@ -201,4 +205,27 @@ public boolean test(ScheduledExecution execution) {
 
     }
 
+    /**
+     * Execution is skipped if the application is not running (either not started or already shutdown).
+     */
+    @ApplicationScoped
```

Comment:
I think that `@Singleton` would be a better match as no client proxy is required here...